### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by wwzz-max** [ChenLiu-1996/CUTS#15]
How to deal with overfitting in model？

**Comment by ChenLiu-1996** [comment]
I have not experienced overfitting in my use cases, and I haven't specifically designed ways to handle overfitting.

However, if you encounter overfitting, you could try some standard techniques:
1. Increasing your training data, either by adding real data or by data augmentation.
2. Weight regularization, for example, increasing weight decay in the optimizer.

**Comment by ChenLiu-1996** [comment]
@wwzz-max Congratulations! I saw your recent published paper https://link.springer.com/article/10.1007/s00371-024-03691-x. I am glad to help your research with this repository.
